tensorboard/uploader/uploader_test.py

```diff
@@ -50,6 +51,19 @@
 from tensorboard.util import test_util as tb_test_util
 
 
+def _create_example_graph(test_attr_size):
```

Comment:
Rename this to _create_example_graph_bytes to make it clear that it'll return `bytes`, not a `GraphDef`.

Reply:
Done.